src/libsync/owncloudtheme.cpp

```diff
@@ -56,5 +56,11 @@ QPixmap ownCloudTheme::wizardHeaderLogo() const
     return QPixmap(hidpiFileName(":/client/theme/colored/wizard_logo.png"));
 }
 
+QString ownCloudTheme::conflictDocumentationLink() const
+{
+    return QString("https://doc.owncloud.org/desktop/%1.%2/conflicts.html")
```

Comment:
Please Use Theme::helpUrl as a base,  and move the function to Theme  and remove the override in ownCloudTheme.

Reply:
Oh, I didn't see `helpUrl()`. Yes, I didn't want oc-branded urls to show up in themes that weren't adjusted for this change. But if `helpUrl` already exists I'll just use that.